### ISSUE TITLE
Feature: support looply, automatically play slides

### ISSUE THREAD

**Issue by litaotao** [damianavila/RISE#141]

**Comment by litaotao** [comment]
hi, there, just find the way to auto, looply play it

just need to config it , add two config key words:

autoSlide: 2000  # auto every 2 seconds


thanks again for creating such an awesome project

**Comment by damianavila** [comment]
> hi, there, just find the way to auto, looply play it

Yes, I know... but you probably needed to hack the source, we should provide an option for that... so let's get this open until we push a config option...

**Comment by litaotao** [comment]
ok, that's would be nice, thanks
